app/src/main/java/org/fossasia/phimpme/editor/fragment/FrameFragment.java

```diff
@@ -298,6 +298,11 @@ public void setHeight(int height) {
     // Asynchronous loading of thumbnails
     private class asyncThumbs extends AsyncTask<Void, Void, Void> {
 
+        @Override
+        protected void onPreExecute() {
+            arrayList=new ArrayList<>();
+        }
+
         @Override
         protected Void doInBackground(Void... params) {
             arrayList = new ArrayList<>();
```

Comment:
@codepoet2017390 You have already initialized this ArrayList in the onPreExecute method then why do it here too?